ambry-account/src/integration-test/java/com/github/ambry/account/MySqlAccountServiceIntegrationTest.java

```diff
@@ -1009,18 +1012,28 @@ public void testLatestVersionSupport() throws Exception {
             testContainer.getName(), DATASET_NAME, version, -1, System.currentTimeMillis(), false,
             DatasetVersionState.READY);
     assertEquals("Mismatch in dataset", expectedDatasetVersionRecord, datasetVersionRecordFromMysql);
+  }
 
+  /**
+   * Test add and get latest version with semantic versioning format.
+   * @throws Exception
+   */
+  @Test
+  public void testLatestSemanticVersionSupport() throws Exception {
```

Comment:
Can you please add your own test cases instead of modifying the existing one? Currently I'm adding some integration test as well. Once it's done, you can refer it and add your test regarding this change as well.